api/routes/settings.ts

```diff
@@ -0,0 +1,70 @@
+import { FastifyPluginCallback } from 'fastify';
+import { Type } from '@sinclair/typebox';
+
+export const settingRoutes: FastifyPluginCallback = (
```

Comment:
```suggestion
export const settingRoutes: FastifyPluginCallbackTypebox = (
```
This means `fastify.put` can infer the Body type from the schema.